src/DynamicMvvm.Uno.WinUI/DynamicMvvm.Uno.WinUI.csproj

```diff
@@ -0,0 +1,73 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.44">
+	<PropertyGroup>
+		<!-- Change the TargetFrameworks depending on which platform you are building on. This avoids errors as it is impossible to build UAP on OSX (MacOS) -->
+		<TargetFrameworks Condition="'$([MSBuild]::IsOsPlatform(OSX))'">netstandard2.0;net6.0-ios</TargetFrameworks>
```

Comment:
@MatFillion Should we support the same platforms as Uno? (including macos, mac catalyst, etc.)

Reply:
There is a issue with Catalyst at the moment to build properly.
Maybe we could confirm with 17.3.3 if it's still happening